liburcu.yaml

```diff
@@ -0,0 +1,92 @@
+package:
+  name: liburcu
+  version: "0.15.3"
+  epoch: 0
+  description: Userspace RCU (read-copy-update) library.
+  copyright:
+    - license: LGPL-2.1
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/urcu/userspace-rcu
+      tag: v${{package.version}}
+      expected-commit: ba5f2bfb6646701a7268c5a03d0454d68b083948
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  # Remove optional examples to reduce package size
+  - runs: |
+      rm -rf ${{targets.destdir}}/usr/share/doc/userspace-rcu/examples
```

Comment:
I think we don't need to do this because we've doc packages that've examples in it and we're moving `/usr/share/doc` to `-doc` package already.

Reply:
Noted, making updates accordingly